### DISCUSSION TITLE
Don't put items added to lists in main starred repo list.

### DISCUSSION THREAD

**Discussion by aw1cks** [community/community#9181] [Other Feature Feedback, Questions, & Ideas]
In my opinion, it doesn't necessarily make sense to couple these together - lists should be treated as discrete from the main 'starred' list.

This would allow the 'starred' section to be used as a generic 'catch-all'/uncategorized section while reducing the clutter.
At the very least, a filter to hide entries already added to a list would be nice.

**Comment by ckosmic** [comment]
I just want a way to group my own similar repos without organizations/projects, and lists requiring repos to be starred makes it look like I'm "liking my own posts" in my followers' feed each time I add one.

**Comment by alex-kim-dev** [comment]
I was just about to request adding a 'not in a list' filter to the main list, but the first suggestion is nice too
